src/utils/create-elements/index.spec.js

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it } from 'vitest';
+import createElements from '.';
+
+describe('create-elements', () => {
+  it('should return an empty array', () => {
+    const html = '';
+    const elements = createElements(html);
+
+    expect(elements).toBeInstanceOf(Array);
+    expect(elements.length).toBe(0);
+  });
+
+  it('should return an array with 2 elements', () => {
+    const text = 'What we have for today?';
+    const html = `
+      <p>
+        <span>Hello</span>
+        <span>World</span>
+      </p>
+      <p>${text}</p>
+    `;
+    const elements = createElements(html);
+
+    expect(elements).toBeInstanceOf(Array);
+    expect(elements.length).toBe(2);
+    expect(elements[1].textContent).toBe(text);
+  });
```

Comment:
Vamos seguir a guideline do Octopost pra testes?
ref: https://github.com/devhatt/octopost/blob/master/STYLEGUIDE.md#testes-unit%C3%A1rios

1. Evitar o uso de should, fica meio repetitivo it should....
2. Pra cada it deve haver um describe antes descrevendo melhor deixando o it mais para o que é esperado que aconteça.